frontend/src/app/shared/scss/_helpers.scss

```diff
@@ -169,3 +169,6 @@
     background-color: $color-grey-070;
   }
 }
+.there--is {
+  display: block !important;
+}
```

Comment:
nu imi place denumirea de '**`there--is`**' , 

mai bine pui '**`has-rating`**'